Assets/Scripts/UI/CoreGameplay/Carousel/Carousel.cs

```diff
@@ -110,7 +111,6 @@ private void InitializeCarousel(Musician currentMusician, Instrument currentInst
         }
 
         // Add the available items
-        //TODO Replace string with a data type
```

Comment:
Is this TODO for later, or does it need to be addressed in this PR? If for later, does a PR exist for it?

Reply:
The TODO was removed as it was already fixed